src/inmanta/doctools.py

```diff
@@ -1,118 +0,0 @@
-"""
```

Comment:
Do we really want to delete this for iso4 as well?

Reply:
Well. I don't see where it's used at the moment, so I don't see the problem with removing it. If we do use it, the CI pipeline will point that out soon.

Reply:
Fair enough